spec/migrations/20180405145642_remove_provider_region_for_google_provider_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_migration
+
+describe RemoveProviderRegionForGoogleProvider do
+  migration_context :up do
+    let(:ems_stub) { migration_stub :ExtManagementSystem }
+
+    ems_stub.create!(:type            => 'ManageIQ::Providers::Google::CloudManager',
```

Comment:
Please wrap these calls into `before` block

Reply:
prefer let before `before` block

Reply:
I meant the `ems_stub.create!` calls. You want to wrap them in a `let` call?